src/interfaces/swig/SGBase.i

```diff
@@ -238,6 +238,10 @@ public void readExternal(java.io.ObjectInput in) throws java.io.IOException, jav
 %}
 
 %feature("nothread") _swig_monkey_patch;
+%feature("docstring", "Adds a Python object (such as a function) \n"
+					  "to a class (method) or to a module. \n"
+					  "The name of the function should not conflict with \n"
+	   				  "another Python object in the same scope") _swig_monkey_patch;
```

Comment:
or what?

Reply:
I think your original function disappears... I should add a check to prevent that from happening actually..

Reply:
I added a check in both functions that raises an error if it destroys a function in the given scope